### DISCUSSION TITLE
✅ 🚀 Feature Request: Support templates as nested content for accordions/tabs/etc

### DISCUSSION THREAD

**Discussion by Nantris** [elementor/elementor#23892] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It would be great to add templates to nested accordion like you can in Essential Addons Elementor

### Describe the solution you'd like

Allow templates in elements

### Additional context

Similar but not the same as #23889

**Comment by nicholaszein** [comment]
Hello @Slapbox!

Thank you for your suggestion! 🙏 

This is already possible to achieve using the [Template widget (Pro)](https://elementor.com/help/template-widget-pro/) inside each nested item. That's the beauty of Nested Elements. They already allow you to add any content inside them. You don't really need a setting in the widget itself to achieve what you want.

I'm closing this request as solved.

Kind regards
